crates/bevy_render/src/render_resource/buffer_vec.rs

```diff
@@ -60,6 +67,12 @@ impl<T: Pod> BufferVec<T> {
         index
     }
 
+    /// Creates a buffer on the [`RenderDevice`](crate::renderer::RenderDevice) with size
```

Comment:
This doesn't create a buffer, it grows an existing one to the specified size unless it's already that size or larger :)

Reply:
>  yeah so it's basically impossible to grow a buffer as that would mean that there would have to be space after the buffer in memory and most of the time there isn't actually space there, the allocator has already used it

- `@cwfitzgerald`

So my comment was wrong, and your original docs are more correct.